conjure-java-server-verifier/src/test/java/com/palantir/conjure/java/verification/server/undertest/UndertowServerUnderTestRule.java

```diff
@@ -42,19 +40,17 @@ protected void before() throws Throwable {
                 UndertowAutoDeserializeService.class, new EchoResourceInvocationHandler());
         Endpoint endpoint = AutoDeserializeServiceEndpoint.of(service);
 
-        RoutingHandler routingHandler = Handlers.routing();
-
-        RoutingRegistry routingRegistry = new ConjureRoutingRegistry(routingHandler);
+        ConjureHandler handler = new ConjureHandler();
         SerializerRegistry serializers = new SerializerRegistry(Serializers.json(), Serializers.cbor());
         HandlerContext context = HandlerContext.builder()
                 .serializerRegistry(serializers)
                 .build();
 
-        endpoint.create(context).register(routingRegistry);
+        endpoint.create(context).register(handler);
```

Comment:
To enable inverting the flow, maybe we could add a method in ConjureHandler such that could be read as 
`handler.register(endpoint.create)`. 

That would enable to write directly, `.setHandler(Handlers.path().addPrefixPath("/test/api", handler.register(endpoint.create(context))` (and even inline everything if we wanted sine no statement above break the flow.

Reply:
We could, however in most cases I expect that the webserver owns (and does not expose) the HandlerContext, and must iterate through all `Endpoint` to build endpoint mappings, so I'm not sure how helpful it will be outside of the simple demo/test case.
I'd like to keep the surface area minimal for the time being, if we find that such a method makes usage easier we can revisit.